### ISSUE TITLE
Use Github Sponsors to support jrnl

### ISSUE THREAD

**Issue by maebert** [jrnl-org/jrnl#861]
## Feature Request

Github just opened up their [Sponsors](https://github.com/sponsors) program. It works similar to Patreon in that people can support the project with recurring donations. Our only fixed costs right now are the domain (~$50/yr), so we don't need deep pockets, but it would be great to see the project being carried by the community.

https://github.com/sponsors/jrnl-org/waitlist

Opening this issue for discussion.

**Comment by micahellison** [comment]
I think it's a nice idea. You brought up Open Collective a while back too, right? I was looking them up and stumbled onto an article of theirs [comparing and contrasting their program with Github Sponsors](https://medium.com/open-collective/on-github-sponsors-5450b63caf7d). Either way, it's nice to have a sense of collective support.

**Comment by maebert** [comment]
We’d actually have to use open collective either way as a fiscal entity (unless you wanted to incorporate a 501(c), which is a bit of an overkill); the advantage of Github Sponsors is that it’s a lot lower friction to donate though as everybody already has an account and often a card on file.

**Comment by dbxnr** [comment]
>  Our only fixed costs right now are the domain (~$50/yr)

Would it be possible to use GitHub for hosting the content itself? It's free although I think quite limited in that it's all client-side.  That doesn't cover the recurring domain registration though..

**Comment by maebert** [comment]
The content is hosted on Github pages already, the cost is just the domain (.sh come at a bit of a premium, but I thought it'd be very fitting).

**Comment by wren** [comment]
I think this is a good idea. I know it's something that's come up occasionally and I'd love to have the project supports its own costs. I've set up open collective (https://opencollective.com/jrnl), and added the sponsor button to the Github repo (up by the stars).

For Github sponsors, there's a waitlist, apparently. I got on the waitlist, so we'll see if/when that happens, but for now, the sponsor button should get us started.

Thanks for suggesting this!
